firestore/tests/system.py

```diff
@@ -729,18 +729,26 @@ def test_collection_group_queries_filters(client, cleanup):
 
     query = (
         client.collection_group(collection_group)
-        .where("__name__", ">=", client.document("a/b"))
-        .where("__name__", "<=", client.document("a/b0"))
+        .where(
+            firestore.field_path.FieldPath.document_id(), ">=", client.document("a/b")
+        )
+        .where(
+            firestore.field_path.FieldPath.document_id(), "<=", client.document("a/b0")
+        )
     )
     snapshots = list(query.stream())
     found = set(snapshot.id for snapshot in snapshots)
     assert found == set(["cg-doc2", "cg-doc3", "cg-doc4"])
 
     query = (
         client.collection_group(collection_group)
-        .where("__name__", ">", client.document("a/b"))
         .where(
-            "__name__", "<", client.document("a/b/{}/cg-doc3".format(collection_group))
+            firestore.field_path.FieldPath.document_id(), ">", client.document("a/b")
```

Comment:
Can we do 
`firestore.field_path.document_id()`
instead of
`firestore.field_path.FieldPath.document_id()`
?
you can explore the exact usecase how this would be access though.

Reply:
as we discussed i am going with firestore.field_path.FieldPath.document_id()